msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/UserNamePasswordParameters.java

```diff
@@ -58,6 +58,13 @@ public class UserNamePasswordParameters implements IAcquireTokenParameters {
      */
     private String tenant;
 
+    /**
+     * Sets the auth scheme for this request.
+     *
+     * For more information about auth schemes, see {@link AuthScheme} and (TODO: link to documentation page)
+     */
+    private AuthScheme authScheme;
+
```

Comment:
There are no tests in this PR. 
The API Review spec has acceptance tests - you should automate as many as possible - https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=/PoPTokensProtocol/PoP_API_In_MSAL.md&version=GBdev&_a=preview


If PR is not ready for review, consider marking it as draft. 

- We should have an test that gets a POP token via ROPC
- The test should also assert that no tokens are stored in MSAL's cache.

Reply:
As of the latest commit mosts of the acceptance tests in that design doc are now implemented (the Mac/Linux tests are missing, as our pipelines only run in Windows environments anyway)